ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/BraveTranslateScript.js

```diff
@@ -22,15 +22,15 @@ Object.defineProperty(window.__firefox__, "$<brave_translate_script>", {
       return document.documentElement.outerText;
     }),
     "checkTranslate": (function() {
-      setTimeout(function() {
-        try {
+      try {
+        if ((cr.googleTranslate.libReady || cr.googleTranslate.finished) && !cr.googleTranslate.readyCallback) {
```

Comment:
This fixes the `Onboarding` button sometimes showing as if it's blended with reader-mode or in the wrong spot. It's because the page isn't ready to be translated yet, so onboarding showed up early.